### PR TITLE
Update plot.py

### PR DESCRIPTION
**Pull request by eurusebr**
I made all the three changes: 
copy-paste plot.box and rename it to plot.box_boundary
add deprecation note to plot.box
add deprecation warning to plot.box

### CODE REVIEW

File: amep/plot.py [hunk 1]
```diff
@@ -914,6 +914,8 @@ def box(axis: mpl.axes.Axes, box_boundary: np.ndarray, **kwargs) -> None:
     -------
     None.
     '''
+    warnings.simplefilter('always', PendingDeprecationWarning)
```
Comment (kay-ro) [inline amep/plot.py:917]: already set in header of python document. please remove here.

**Review by kay-ro — CHANGES_REQUESTED** [review]

File: amep/plot.py (hunk 1 above)
Comment (eurusebr) [reply amep/plot.py]: thanks, I made the change as requested :)

**Review by kay-ro — APPROVED** [review]
looks good. thank you :)

File: amep/plot.py (hunk 1 above)
Comment (kay-ro) [reply amep/plot.py]: thank you :)

**Pull request merged**